docs/content/general/tutorial.md

```diff
@@ -1564,4 +1564,61 @@ EOF
 >* v1.20.6
 >* v1.21.0
 
-This ends our tutorial on how to implement complex multi-tenancy and policy-driven scenarios with Capsule. As we improve it, more use cases about multi-tenancy, policy admission control, and cluster governance will be covered in the future. Stay tuned!
\ No newline at end of file
+## Limiting the number of resources deployed in a Tenant
```

Comment:
To make it consistent with other sections of the tutorial, please change the title to "Assign custom resources quota"

Reply:
Please, move this section just after the `Create Custom Resources` section so there is a smooth flow between the two.